pkg/common/proto/agent.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+option go_package = ".;pb";
+
+service AgentService{
+    rpc PerformFantasyActions(FantasyLandscape) returns (FantasySelections) {}
+}
+
+message FantasyLandscape{
+    int32 matchNumber = 1;
```

Comment:
nit: Probably makes sense to make these unsigned integers because we don't need the signed behavior.

Also lets be ambitious and assume this is gonna pop off enough to overflow a 32 bit integer :laughing: